runtime/app-runtime.js

```diff
@@ -585,32 +580,14 @@ AppRuntime.prototype.resetServices = function resetServices (options) {
   return Promise.all(promises)
 }
 
-AppRuntime.prototype.appPause = function appPause (appId) {
-  logger.info('Pausing resources of app', appId)
-  var promises = [
-    this.component.light.stopSoundByAppId(appId),
-    this.multimediaMethod('pause', [ appId ])
-  ]
-  return Promise.all(promises)
-    .catch(err => logger.error('Unexpected error on pausing resources of app', appId, err.stack))
-}
-
-AppRuntime.prototype.appGC = function appGC (appId) {
+AppRuntime.prototype.appDidExit = function appDidExit (appId) {
   logger.info('Collecting resources of app', appId)
   var promises = [
-    this.component.light.stopByAppId(appId),
-    this.component.light.stopSoundByAppId(appId),
```

Comment:
add light. appDidExit here ?

Reply:
`runtime.componentsInvoke` would iterate over components and invoke expected methods e.g. `appDidExit`  immediately.